src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-daemon-details/rgw-daemon-details.component.html

```diff
@@ -21,6 +22,8 @@
       </ng-template>
     </li>
     <li ngbNavItem
+        cdStatefulTab
+        stateId="performance-details"
```

Comment:
forgot to change these
```suggestion
        ngbNavItem="performance-details"
```